lib/jsonapi_compliable/responders.rb

```diff
@@ -0,0 +1,19 @@
+# If you're using Rails + responders gem to get respond_with
+module JsonapiCompliable
+  module Responders
+    extend ActiveSupport::Concern
+
+    included do
+      include ActionController::MimeResponds
+      respond_to :json, :jsonapi, :xml, :api_json
```

Comment:
What is `api_json`?  Also should probably make this default list configurable via `JsonapiCompliable.configure do |config|`, as not everyone will want e.g. xml.